pkg/router/template/configmanager/haproxy/map.go

```diff
@@ -0,0 +1,209 @@
+package haproxy
```

Comment:
Give this its own unit tests.

Reply:
there's a `map_test.go` in there. Or did you mean something else?